dev/README_RELEASE_AIRFLOW_UPGRADE_CHECK.md

```diff
@@ -0,0 +1,642 @@
+- [Verify the release candidate by PMCs](#verify-the-release-candidate-by-pmcs)
+  - [SVN check](#svn-check)
+  - [Licence check](#licence-check)
+  - [Signature check](#signature-check)
+  - [SHA512 sum check](#sha512-sum-check)
+- [Verify release candidates by Contributors](#verify-release-candidates-by-contributors)
+- [Publish the final release](#publish-the-final-release)
+  - [Summarize the voting for the release](#summarize-the-voting-for-the-release)
+  - [Publish release to SVN](#publish-release-to-svn)
+  - [Prepare PyPI "release" packages](#prepare-pypi-release-packages)
+  - [Notify developers of release](#notify-developers-of-release)
+  - [Update Announcements page](#update-announcements-page)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+You can find the prerequisites to release Apache Airflow in [README.md](README.md). This document
+details the steps for releasing apache-airflow-upgrade-check package.
+
+# Prepare the Apache Airflow Upgrade Check Package RC
+
+## Pre-requisites
+
+- Install `setuptools-build-subpackage` v0.1.0 by running:
+
+  ```shell
+  pip install https://github.com/ashb/setuptools-build-subpackage/archive/master.zip
+  ```
+
+## Build RC artifacts
+
+The Release Candidate artifacts we vote upon should be the exact ones we vote against,
+without any modification than renaming – i.e. the contents of the files must be
+the same between voted release candidate and final release.
+Because of this the version in the built artifacts that will become the
+official Apache releases must not include the rcN suffix.
+
+- Set environment variables
+
+    ```shell script
+    # Set Version
+    export VERSION=1.3.0rc1
+
+
+    # Set AIRFLOW_REPO_ROOT to the path of your git repo
+    export AIRFLOW_REPO_ROOT=$(pwd)
+
+
+    # Example after cloning
+    git clone https://github.com/apache/airflow.git airflow
+    cd airflow
+    export AIRFLOW_REPO_ROOT=$(pwd)
+    ```
+
+- Checkout `v1-10-stable` branch:
+
+    ```shell
+    git checkout v1-10-stable
+    ```
+
+- Set your version to 1.3.0 in `airflow/upgrade/version.py` (without the RC tag)
+- Commit the version change.
+
+- Tag your release
+
+    ```shell script
+    git tag -s upgrade-check/${VERSION}
+    ```
+
+- Clean the checkout: the sdist step below will
+
+    ```shell script
+    git clean -fxd
+    ```
+
+- Tarball the repo
+
+    ```shell script
+    git archive  upgrade-check/${VERSION} --prefix=apache-airflow-upgrade-check-${VERSION%rc?}/ \
+        -o apache-airflow-upgrade-check-${VERSION}-source.tar.gz \
+        airflow/upgrade NOTICE LICENSE licenses/
+    ```
+
+
+- Generate sdist
+
+    NOTE: Make sure your checkout is clean at this stage - any untracked or changed files will otherwise be included
+     in the file produced.
+
+    ```shell script
+    python -m setuptools_build_subpackage --subpackage-folder airflow/upgrade \
+        egg_info sdist --license-template ./LICENSE bdist_wheel
```

Comment:
I don't think you need to run `egg_info` command manually, `sdist` will do that automatically I thought

Reply:
Yeah, `egg_info` is only needed when preparing the pypi snapshot so that we can pass `--tag-build` option, but we shouldn't need it here.

Reply:
Updated and rebased